### PR TITLE
Adjustments to examples/softirq-latency-net-rx

### PR DESCRIPTION
**Pull request by netoptimizer**
Making some adjustments to examples/softirq-latency-net-rx, prior to using this in prod.

(Related to PERF-1546)

### CODE REVIEW

File: examples/softirq-latency-net-rx.bpf.c [hunk 1]
```diff
@@ -86,7 +96,7 @@ int BPF_PROG(softirq_raise, unsigned int vec_nr)
 SEC("tp_btf/softirq_entry")
 int BPF_PROG(softirq_entry, unsigned int vec_nr)
 {
-    u64 delta_us, *raise_ts_ptr, *existing_entry_ts_ptr, *serviced_total_ptr, ts;
+    u64 delta_ns, *raise_ts_ptr,  *serviced_total_ptr, ts;
```
Comment (bobrik) [inline examples/softirq-latency-net-rx.bpf.c:99]: The failed `clang-format` build has the diff that needs to be applied to make it happy.

File: examples/softirq-latency-net-rx.bpf.c [hunk 2]
```diff
@@ -4,14 +4,21 @@
...
     if (vec_nr != NET_RX_SOFTIRQ) {                                                                                    \
         return 0;                                                                                                      \
     }
 
+/* This provide easy way to disable measuring 'runtime'.
+ * This avoids hooking 'softirq_exit' as it can be expensive and for NET_RX
+ * this isn't the right hook as runtime is affected by NAPI packet bulking.
+ */
+#define CONFIG_MEASURE_RUNTIME	1
+//#undef CONFIG_MEASURE_RUNTIME
```
Comment (bobrik) [inline examples/softirq-latency-net-rx.bpf.c:20]: Let's drop this line. I think one can simply comment the line above to achieve the same outcome.

File: examples/softirq-latency-net-rx.bpf.c [hunk 3]
```diff
@@ -4,14 +4,21 @@
 
 // Loosely based on https://github.com/xdp-project/xdp-project/blob/master/areas/latency/softirq_net_latency.bt
 
-// 20 buckets for latency, max range is 0.5s .. 1.0s
-#define MAX_LATENCY_SLOT 21
+// 30 buckets for latency, max range is 0.5s .. 1.0s
+#define MAX_LATENCY_SLOT 31
```
Comment (bobrik) [inline examples/softirq-latency-net-rx.bpf.c:8]: Is nanosecond precision worth 10 extra buckets? Looking at the output here, perhaps the answer is yes:

* https://bugzilla.redhat.com/show_bug.cgi?id=1795049

Comment (bobrik) [reply examples/softirq-latency-net-rx.bpf.c:8]: You have this info in the commit already.

File: examples/softirq-latency-net-rx.bpf.c (hunk 1 above)
Comment (netoptimizer) [reply examples/softirq-latency-net-rx.bpf.c]: Used clang-format and fixed this and others

**Review by bobrik — APPROVED** [review]

**Pull request merged**